web/styles/app_variables.css

```diff
@@ -799,6 +799,37 @@
     --color-compose-control-button-background-hover: hsl(0deg 0% 0% / 5%);
     --color-compose-formatting-button-divider: hsl(0deg 0% 75%);
     --color-compose-focus-ring: var(--color-outline-focus);
+    --color-compose-scroll-icon: #7f7f7f;
```

Comment:
Why is this color not using HSL?

Reply:
That was how Vlad had it in his mock. But a shade of gray is not hard to convert to HSL.